### ISSUE TITLE
Malformed bower file

### ISSUE THREAD

**Issue by Manduro** [twinssbc/Ionic-Calendar#1]
Thanks for releasing your Ionic version.

I've noticed a missing comma in the bower file. This causes bower to throw an error on installation. Easy fix: add a comma behind the ionic dependency ;)

```
    "dependencies": {
        "angular": "~1.2",
        "ionic": "~1.1.1"
        "bootstrap": "~3.0.3"
    },
```

**Comment by twinssbc** [comment]
@Manduro actually I haven't registered this lib on bower, I will  do it during the weekend

**Comment by twinssbc** [comment]
@Manduro actually I haven't registered this lib on bower, I will  do it during the weekend

**Comment by twinssbc** [comment]
@Manduro actually I haven't registered this lib on bower, I will  do it during the weekend

**Comment by Manduro** [comment]
@twinssbc Yes I noticed, that's fine. This just fixes the bower file so in the meantime it can be added using `bower install twinssbc/Ionic-calendar`.

**Comment by Manduro** [comment]
Fixed with #2.
